assets/scss/colors.module.scss

```diff
@@ -1,11 +1,12 @@
+// I had to prefix the color with "rl" because some of the base color names were protected somehow
```

Comment:
updated the colors.module.scss with unique names... because the basic names we protected somehow.